EasyPost.EasyVCR/Statics.cs

```diff
@@ -0,0 +1,63 @@
+using System.Collections.Generic;
+
+namespace EasyPost.EasyVCR
+{
+    /// <summary>
+    ///     Static values used as defaults for EasyVCR.
+    /// </summary>
+    internal static class Statics
+    {
+        /// <summary>
+        ///     Default list of headers to censor in the cassettes.
+        /// </summary>
+        internal static List<string> DefaultCredentialHeadersToHide => new()
+        {
+            {
+                "Authorization"
+            }
+        };
+
+        /// <summary>
+        ///     Default list of parameters to censor in the cassettes.
+        /// </summary>
+        internal static List<string> DefaultCredentialParametersToHide => new()
```

Comment:
When searching for creds to hide, do we lowercase them on matching? Otherwise this will fail with differing casings.

Reply:
Good catch.